doc/sphinx/proof-engine/vernacular-commands.rst

```diff
@@ -1227,3 +1227,39 @@ Exposing constants to OCaml libraries
 
    Due to its internal nature, this command is not for general use. It is meant
    to appear only in standard libraries and in support libraries of plug-ins.
+
+   As a special case, when the first segment of :n:`@qualid__2` is :g:`kernel`,
+   the constant is exposed to the kernel. For instance, the `Int63` module
```

Comment:
Why doesn't the kernel reuse the standard mechanism? Because that would have increased the TCB?

Reply:
Indeed. Coqlib is not in the kernel.